packages/framework-provider-kubernetes/src/services/event-registry.ts

```diff
@@ -58,7 +58,7 @@ export class EventRegistry {
     if (result.length <= 0) {
       return null
     }
-    return result[0]
+    return result[result.length - 1]
```

Comment:
Did you manage to test if this is the right order of getting items?

Reply:
yes!! the readmodels reductions were failing because of this. I checked using logs messages that the array was ordered from old to new so the newest snapshot is the latest in the array! 🤓